test/CMakeLists.txt

```diff
@@ -218,6 +220,7 @@ if( iodaconv_bufr_ENABLED )
     testoutput/bufr_read_wmo_radiosonde.nc
     testoutput/NC005031.nc
```

Comment:
Did you add lines 221-222 in an earlier PR? Whoever did, there must be  matching testinput/*.bufr_d and testinput/*.yaml for them. This is not part of your current PR, but I spotted it, so we clean it or fix it.

Reply:
I looked around, this is a non-problem, it has matching testinput/* files, but I didn't see them at first.

Reply:
@rmclaren , @emilyhcliu , lines 221-222  (and the matching testinput lines and tests) are no longer needed, I'll open a PR to clean this up.